app/auth/mutations/signup.test.ts

```diff
@@ -0,0 +1,65 @@
+import { hash256, Ctx } from "blitz"
+import db from "db"
+import signup from "../queries/signup"
+
+beforeEach(async () => {
+  await db.$reset()
+})
+
+describe("signup queue", () => {
+  it("does throw error if token doesn't exist", async () => {
+    // use not existing token
+    await expect(signup({ token: "plain-token" }, {} as Ctx)).resolves.toBe(null)
+  })
+
+  it("does throw error if token expired", async () => {
+    // create expired test token
+    const generatedToken = "plain-token"
+    const date = new Date()
+    date.setHours(date.getHours() - 5)
+    const goodPassword = "good-password"
+    await db.tokenMailVerification.create({
+      data: {
+        type: "LOGIN_VERIFY",
+        expiresAt: date,
+        hashedToken: hash256(generatedToken),
+        sentTo: "user@email.com",
+        name: "user",
+        hashedPassword: hash256(goodPassword),
+      },
+    })
+    // use expired token
+    expect(signup({ token: generatedToken }, {} as Ctx)).resolves.toBe(null)
```

Comment:
why should it throw NULL and not an error?

Reply:
see first comment ;)